### ISSUE TITLE
shiny::observeEvent with multiple events followed by if statement

### ISSUE THREAD

**Issue by SchalkWBurger** [jimhester/lintr#359]
To reproduce the error follow the following structure:
```
  shiny::observeEvent({ 
    input1
    input2
    input3
  }, {
    if (condition) > 0 &&
      length(condition) > 0 &&
     {
      shinyjs::show(id = "instrument")
    } else {
      shiny::showNotification(
        ui = "error message",
        type = "error"
      )
      shinyjs::hide(id = "instrument")
    }
  })
```

The lintr breaks in line 5 ``}, {`` with the following message: _Closing curly-braces should always be on their own line, unless it's followed by an else_.

I used #nolint start and #nolint end to fix the problem for now.

**Comment by jimhester** [comment]
The code you showed is not actually valid R code, but tweaking it to do what I think you meant has no lints with the current master.

```r
shiny::observeEvent({
  input1
  input2
  input3
}, {
  if (condition > 0 &&
    length(condition) > 0) {
    shinyjs::show(id = "instrument")
  } else {
    shiny::showNotification(
      ui = "error message",
      type = "error"
    )
    shinyjs::hide(id = "instrument")
  }
})
```
